test_data/definitions.yaml

```diff
@@ -24,9 +24,11 @@ urls: ['http://support.microsoft.com/kb//214653']
 ---
 name: CurrentControlSet
 doc: The control set the system is currently using.
-collectors:
-- collector_type: REGISTRY_VALUE
-  args: {path_list: ['HKEY_LOCAL_MACHINE\SYSTEM\Select\Current']}
+sources:
+- type: REGISTRY_VALUE
+  attributes:
+    key_value_pairs:
+      - {key: 'HKEY_LOCAL_MACHINE\SYSTEM\Select', value: 'Current'}
```

Comment:
I think the value syntax there is too confusing, mixing keys and values didn't work out well the first time.

MSFT uses empty string for the default value:
https://msdn.microsoft.com/en-us/library/microsoft.win32.registrykey.getvaluenames(v=vs.110).aspx

I think we should just do that.

{key: 'CurrentVersion\Run\*' value: ''}  #get default value